server/src/uuid/model/taxonomy_term/mod.rs

```diff
@@ -236,3 +242,120 @@ impl TaxonomyTerm {
         typename.to_case(Case::Camel)
     }
 }
+
+#[derive(Error, Debug)]
+pub enum TaxonomyTermSetNameAndDescriptionError {
+    #[error(
+        "Taxonomy Term name and description cannot be set because of a database error: {inner:?}."
+    )]
+    DatabaseError { inner: sqlx::Error },
+    #[error(
+        "Taxonomy Term name and description cannot be set because of an event error: {inner:?}."
+    )]
+    EventError { inner: EventError },
+}
+
+impl From<sqlx::Error> for TaxonomyTermSetNameAndDescriptionError {
+    fn from(inner: sqlx::Error) -> Self {
+        Self::DatabaseError { inner }
+    }
+}
+
+impl From<EventError> for TaxonomyTermSetNameAndDescriptionError {
+    fn from(error: EventError) -> Self {
+        match error {
+            EventError::DatabaseError { inner } => inner.into(),
+            inner => Self::EventError { inner },
+        }
+    }
+}
+
+impl From<TaxonomyTermSetNameAndDescriptionError> for operation::Error {
+    fn from(error: TaxonomyTermSetNameAndDescriptionError) -> Self {
+        match error {
+            TaxonomyTermSetNameAndDescriptionError::DatabaseError {
+                inner: sqlx::Error::RowNotFound,
+            } => operation::Error::NotFoundError,
```

Comment:
Should be a UserInputError: "term with term_id does not exist"

Reply:
alright, done in 3f20e00